src/appengine/private/components/upload-testcase/upload-form-simplified.html

```diff
@@ -0,0 +1,414 @@
+        <template is="dom-if" if="[[shouldShowIssueField()]]">
+            <paper-input value="{{uploadParams.issue}}" label="Issue ID" title="Issue ID in the associated project's tracker to update for this test case."></paper-input>
+        </template>
+      </div>
+      <template is="dom-if" if="[[uploadParams.testcaseId]]">
+        <span class="reupload">
+          If you don't choose a file, the testcase from <a href="/testcase-detail/[[uploadParams.testcaseId]]">[[uploadParams.testcaseId]]</a> will be used.
+        </span>
+      </template>
+      <div class="inline">
+        <div>
+          <p>Test case file or archive. If an archive is uploaded, a file containing 'fuzz-', 'run', 'index.', or 'crash.' in its name will be treated as the test case and the rest as resources.</p>
+          <paper-input value="{{uploadParams.file}}" no-label-float always-float-label$="false" type="file" id="file" label="File" title="Test case file or archive. If an archive is uploaded, a file containing &quot;fuzz-&quot;, &quot;run&quot;, &quot;index.&quot;, or &quot;crash.&quot; will be treated as the test case and the rest as resources."></paper-input>
+        </div>
+        <template is="dom-if" if="[[shouldShowFiletypePicker(uploadParams.file)]]">
+          <paper-dropdown-menu
+              class="dropdown-filter"
+              no-animations="true"
+              label="Teset case file type"
+              title="html, js, or pdf"
+              noink>
+            <paper-listbox
+                slot="dropdown-content"
+                selected="{{uploadParams.fileType}}"
+                fallback-selection="html"
+                attr-for-selected="val">
+              <paper-item on-click="submit" val="html">html</paper-item>
+              <paper-item on-click="submit" val="js">js</paper-item>
+              <paper-item on-click="submit" val="pdf">pdf</paper-item>
+            </paper-listbox>
+          </paper-dropdown-menu>
+        </template>
+      </div>
+      <template is="dom-if" if="[[showWarning(uploadParams.file, uploadParams.fileType, uploadParams.platform)]]">
+        <div class="warning">
+          <b>WARNING</b>: ClusterFuzz doesn't support untrusted workloads.<br>
+          Only upload testcases that you trust enough to run on your own
+          machine. Note that some jobs do not use the Chrome sandbox, the
+          following jobs do:<br>
+          [[formatSandboxedJobs(fieldValues.sandboxedJobs)]]
+        </div>
+      </template>
+      <div class="inline wide">
+        <paper-dropdown-menu
+            class="dropdown-filter"
+            no-animations="true"
+            label="Platform"
+            title="Linux, Windows, Mac, or Android"
+            noink>
+          <paper-listbox
+              slot="dropdown-content"
+              selected="{{uploadParams.platform}}"
+              attr-for-selected="val">
+            <paper-item on-click="submit" val="Linux">Linux</paper-item>
```

Comment:
did you mean to have on-click="submit" here?

Reply:
SO, no. and i could have sworn that removing it made it behave weirdly, but ive just tested TWICE and it works great c: